apps/cookbook/src/app/examples/link-example/examples/trailing-icon.ts

```diff
@@ -0,0 +1,24 @@
+import { Component } from '@angular/core';
+
+const config = {
+  selector: 'cookbook-link-example-trailing-icon',
+  template: `<a class="trailing-icon-example" target="_blank" href="https://github.com/kirbydesign/designsystem">Trailing icon</a>`,
+  style: `.trailing-icon-example
+{
+@include utils.trailing-icon("navigation")
```

Comment:
Consider adding the @use statement at the top, so the consumer knows where to import it from...

Reply:
Good idea!